### ISSUE TITLE
.NET 8 as a supported runtime from a source code repository

### ISSUE THREAD

**Issue by rshekhtm** [aws/apprunner-roadmap#239]
**Community Note**
* Please vote on this issue by adding a 👍 reaction to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do * not help prioritize the request
If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Tell us about your request**
.NET 8 is now a fully supported runtime for Lambda, but not App Runner. You can do it via containers, but I am deploying code via a source code repository and that is currently not supported.

**Describe alternatives you've considered**
Can do it via ECR, but that is extra complexity I'd rather not deal with.

**Comment by jwerre** [comment]
Can't you just make a Docker image for your app: https://learn.microsoft.com/en-us/dotnet/core/docker/build-container?tabs=windows&pivots=dotnet-8-0

**Comment by rshekhtm** [comment]
> Can't you just make a Docker image for your app: https://learn.microsoft.com/en-us/dotnet/core/docker/build-container?tabs=windows&pivots=dotnet-8-0

@jwerre - I can, and I called this out as a workaround in the issue. I am specifically asking for source code repo option support so this workaround is not needed. .NET 8 is the latest LTS version and should have support similar to .NET 6, especially given v6 will be EOL this year.

**Comment by EricRuxton** [comment]
1 year later and this is still an "issue". A shame really because Q assured me multiple times that .Net 8 was a supported runtime for AppRunner.

**Comment by rshekhtm** [comment]
Yeah, unfortunately it's pretty clear by now that AppRunner is in maintenance mode, so I'm not holding my breath for any changes at this point.
